src/app/team.tsx

```diff
@@ -42,6 +42,7 @@ const teamMemberDataRow2 = [
 ];
 
 const Team = () => (
+<body>    
```

Comment:
This is technically already in a body tag. Although Chrome autocorrects this, it could have adverse side affects. fyi

Reply:
gotcha,have you seen the actual page?
the first row is blocking the navigation bar